Description: disable valgrind tests
 valgrind is not supported on certain architectures. This patch disables the valgrind
 tests on these architectures so that a build can succeed nonetheless.
Author: Simon Elsbrock
Forwarded: https://github.com/stedolan/jq/pull/196
Index: jq/tests/run
===================================================================
--- jq.orig/tests/run	2014-02-18 08:06:48.000000000 +0000
+++ jq/tests/run	2014-02-18 08:07:50.372781837 +0000
@@ -1,3 +1,3 @@
 #!/bin/sh
-
-cat $@ | valgrind --error-exitcode=1 -q --leak-check=full ./jq --run-tests
\ No newline at end of file
+uname -m | egrep -q 'armel|mips|mipsel|powerpc|s390x' && exit 0
+cat $@ | valgrind --error-exitcode=1 -q --leak-check=full ./jq --run-tests
